ventilator_websocket.py

```diff
@@ -47,14 +47,18 @@ def run(self, name):
 
         while True:
             json_msg = self.ws.recv()
-            msg = json.loads(json_msg)
-            if msg['type'] == "ping":
-                reply = {'type': 'ping'}
-                self.send_msg(reply)
-            elif msg['type'] == "pub":
-                payload = msg['message']
-                if payload['type'] == "setting":
-                    self.handle_settings(payload)
+            print("Websockets msg {}", json_msg)
```

Comment:
Is this print necessary? also you need to call "Websockets msg {}".format(json_msg) for the message to be fromatted into the string in place of the replacement field.

Reply:
You're right, it was for debugging, I forgot to remove it